### PR TITLE
Use a more compact JSON with verbs generated from wordnet

### PR DESCRIPTION
**Pull request by satazor**
The wordnet package is large and its db folder is 28mb in size.
Because this package will be used in a GitHub action, we want the clone process to be as fast as possible.
Having that said, this commit adds a script that generates a very small <= 200KB file with just the English verbs.

### CODE REVIEW

File: bin/update-wordnet-verbs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+/**
+ * Module dependencies.
+ */
+
+const fs = require('fs');
+const wordnet = require('wordnet');
+
+/**
+ * Generates a compact JSON file with all english verbs from wordnet.
```
Comment (diogotorres97) [inline bin/update-wordnet-verbs:13]:
```suggestion
 * Generates a compact JSON file with all english verbs from `wordnet` lib.
```

File: bin/update-wordnet-verbs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ */
+
+const updateVerbs = async () => {
+  console.log(`Finding verbs..`);
+
+  await wordnet.init();
+
+  const verbs = {};
```
Comment (diogotorres97) [inline bin/update-wordnet-verbs:21]: Can you add a comment here explaining that we're storing the verbs as an object `{ "name": true}` so we can use it as a map (more performant comparing to an array)?

File: bin/update-wordnet-verbs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    verbs[word] = true;
+    verbsCount += 1;
+
+    if (verbsCount % 1000 === 0) {
+      console.log(`Found ${verbsCount} verbs`);
+    }
+  }
+
+  console.log(`Writing src/wordnet-verbs.json with a total of ${verbsCount} verbs..`);
```
Comment (diogotorres97) [inline bin/update-wordnet-verbs:41]: Convert file path to a constant `src/wordnet-verbs.json`

File: src/verb-plugin.js [hunk 4]
```diff
@@ -4,65 +4,40 @@
...
-  } catch (err) {
-    return whitelistedVerbs.includes(word);
-  }
-
-  const verb = definitions.find(({ meta }) => meta.synsetType === 'verb');
-
-  return !!verb;
+const allVerbs = {
+  ...wordnetVerbs,
+  // Whitelist additional verbs wordnet does not contain.
```
Comment (diogotorres97) [inline src/verb-plugin.js:16]:
```suggestion
  // Whitelist additional verbs `wordnet` does not contain.
```

Comment (diogotorres97) [reply src/verb-plugin.js:16]: Should we extract this verbs to a file?

File: src/verb-plugin.js [hunk 5]
```diff
@@ -4,65 +4,40 @@
...
+const startsWithVerbInSimplePresent = phrase => {
   const words = phrase.split(/\s/);
   const firstWord = words[0].toLowerCase();
 
-  // Load dictionary just once.
-  await (wordnetInitPromise = wordnetInitPromise ?? wordnet.init());
-
   // Lookup the verb in the dictionary.
-  let valid = await isVerb(firstWord);
+  let valid = !!allVerbs[firstWord];
```
Comment (diogotorres97) [inline src/verb-plugin.js:40]: I would keep the `isVerb` function to avoid doing `!!` all the time.

**Review by diogotorres97 — CHANGES_REQUESTED** [review]

File: src/verb-plugin.js (hunk 4 above)
Comment (diogotorres97) [reply src/verb-plugin.js:16]: yes

File: bin/update-wordnet-verbs (hunk 2 above)
Comment (satazor) [reply bin/update-wordnet-verbs:30]: Added a comment on the doc block of the function.

File: src/verb-plugin.js [hunk 6]
```diff
@@ -4,65 +4,32 @@
...
-    definitions = await wordnet.lookup(word);
-  } catch (err) {
-    return whitelistedVerbs.includes(word);
-  }
-
-  const verb = definitions.find(({ meta }) => meta.synsetType === 'verb');
-
-  return !!verb;
-};
+const isVerb = word => !!allVerbs[word] || !!extraVerbs[word];
```
Comment (diogotorres97) [inline src/verb-plugin.js:21]:
```suggestion
const isVerb = word => !!wordnetVerbs[word] || !!extraVerbs[word];
```

or 

```suggestion
const isVerb = word => !!allVerbs[word];
```

Not a mix 😄

**Review by diogotorres97 — APPROVED** [review]

**Pull request merged**